src/NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestMaxBodySizeLayoutRenderer.cs

```diff
@@ -0,0 +1,51 @@
+﻿#if ASP_NET_CORE3
+using System.Text;
+using Microsoft.AspNetCore.Http.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Enums;
+using NLog.Web.Internal;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// The maximum request body size for a single request
+    /// Uses IHttpMaxRequestBodySizeFeature
+    /// 
+    /// CanRead indicates whether MaxRequestBodySize is read-only.
+    /// If true, this could mean that the request body has already been read from or that UpgradeAsync() was called.
+    /// 
+    /// MaxBodyRequestSize is the maximum allowed size of the current request body in bytes.
+    /// When set to null, the maximum request body size is unlimited.
+    /// This cannot be modified after the reading the request body has started.
+    /// This limit does not affect upgraded connections which are always unlimited.
+    /// </summary>
+    /// <remarks>
+    /// <code>${aspnet-request-max-body-size:Property=IsReadOnly}</code>
+    /// <code>${aspnet-request-max-body-size:Property=MaxBodyRequestSize}</code>
+    /// </remarks>
+    [LayoutRenderer("aspnet-request-max-body-size")]
+    public class AspNetRequestMaxBodySizeLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
When is this useful ?

Reply:
will investigate this one and the ones below further.

Reply:
the Microsoft docs on this are unclear how useful so removed